recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,4 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - '10.12'                # [osx]
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.15"                # [osx and x86_64]
```

Comment:
```suggestion
  - "10.12"                # [osx and x86_64]
```

Reply:
Just to confirm, the deployment target and the SDK do not have to be the same thing?

I think i'm understanding this, but I think I can also make a PR to elaborate on the docs.